cloud-sql-proxy/test_cloud_sql.py

```diff
@@ -16,21 +17,27 @@ def setUpClass(cls):
         super().setUpClass()
         _, region, _ = cls.run_command(
             "gcloud config get-value compute/region")
-        cls.REGION = region.strip() or "global"
+        _, zone, _ = cls.run_command("gcloud config get-value compute/zone")
+        cls.REGION = region.strip() or zone.strip()[:-2]
         _, project, _ = cls.run_command("gcloud config get-value project")
         project = project.strip()
         cls.PROJECT_METADATA = '{}:{}'.format(project, cls.REGION)
 
     def setUp(self):
         super().setUp()
-        self.DB_NAME = "test-cloud-sql-{}".format(self.datetime_str())
-        ret_code, stdout, stderr = self.run_command(
-            'gcloud sql instances create {} --region {}'.format(
-                self.DB_NAME, self.REGION))
+        self.DB_NAME = "test-cloud-sql-{}-{}".format(self.datetime_str(),
+                                                     self.random_str())
+        cmd = 'gcloud sql instances create {} --region {} {}'.format(
+            self.DB_NAME, self.REGION, "--async --format=json")
+        ret_code, stdout, stderr = self.run_command(cmd)
+        operation_id = json.loads(stdout.strip())['name']
+        cmd = 'gcloud sql operations wait {} --timeout=600'.format(
+            operation_id)
+        ret_code, stdout, stderr = self.run_command(cmd)
         self.assertEqual(
-            ret_code, 0,
-            "Failed to create sql instance {}. Last error: {}".format(
-                self.DB_NAME, stderr))
+            ret_code, 0, "Failed to create sql instance {}.".format(
+                self.DB_NAME,
```

Comment:
This format() has 2 args, but only 1 placeholder {}, why?

Reply:
Fixed